core/src/main/resources/reference.conf

```diff
@@ -169,8 +169,9 @@ system {
     number-of-attempts = 5
 
     # I/O Commands that are more than `command-backpressure-staleness` old at the time they begin to be processed will
-    # trigger I/O backpressure.
-    command-backpressure-staleness = 5 seconds
+    # trigger I/O backpressure. Keep this above gcs.max-batch-duration to avoid unwanted competition with that deadline
+    # and consequent excessive backpressure.
```

Comment:
TOL: I wonder if we should build a config sanity checker than runs on start-up and issues warnings if it sees things that we think are a bad idea based on our accumulated knowledge.